frontend/src/ui/BaseLayout/BaseLayout.tsx

```diff
@@ -29,12 +30,15 @@ export const BaseLayout = ({ children }: { children: React.ReactNode }) => {
       >
         {children}
       </Main>
-      <ContentSidebar
-        expanded={contentBarExpanded}
-        toggleContentBar={() => setContentBarExpanded((prevState) => !prevState)}
-      >
-        <div style={{ color: 'black' }}>Right Sidebar</div>
-      </ContentSidebar>
+      {
+        pathname.includes('/tasks') &&
+        <ContentSidebar
+          expanded={contentBarExpanded}
+          toggleContentBar={() => setContentBarExpanded((prevState) => !prevState)}
+        >
+            <WorkSummaryPanel contentSidebarExpanded={contentBarExpanded}/>
+        </ContentSidebar>
+      }
```

Comment:
We think we might use the panel for the project planner tool with some data that is similar to what is here. But I like the idea of moving this to a layout for the page rather than my shameless hack ;)